tests/helpers.py

```diff
@@ -854,6 +858,99 @@ def create_pods(pvc_objs, pod_factory, interface, pods_for_rwx=1, status=""):
     return pod_objs
 
 
+def create_build_from_docker_image(
+        image_name,
+        install_package,
+        namespace,
+        source_image='centos',
+        source_image_label='latest'
```

Comment:
why dont use ubi8 as default image ?
it is much smaller

Reply:
The functionality is to make a build by passing a docker file. I believe it looks for the base image from the local image stream. I don't see a standard ubi8 image available.

`error: unable to locate any images in image streams, local docker images with name "ubi:8"`

But this is a valid point, I'll try to see if there is a leaner image available. I'll make that change as part of a different PR.

Reply:
currect, i am using it but it located at : registry.access.redhat.com/ubi8